docs/Tutorial_DAMP.ipynb

```diff
@@ -0,0 +1,592 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #36.        chunks_range = naive_get_range_damp(split_idx - excl_zone + m, m, start=0)
</pre><p><strong>[suggestion]</strong></p><p>We might use the name <code>chunks_geometric_range</code> as the range of chunks are following the geometric series with the initial value of next_pow2(m) (the coefficient) and the ratio of 2 . It is better to set this initial value and pass it to the function <code>naive_get_range_dump</code> because the same initial value will be used later to compute <code>expected_last_chunk_size</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/920/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Going to ignore this for now.